### DISCUSSION TITLE
按照文档的方式引入的组件，但一直提示“Error: Component is not found in path”

### DISCUSSION THREAD

**Discussion by wilde-1206** [TaleLin/lin-ui#1392] [求助]
"qn-loadmore":"/components/lin-ui/loadmore/index"

路径应该没问题，我自己自定义组件正常使用，就是引用的lin-ui出现这个提示，然后模拟器一片空白，但手机预览可以正常看，请问大佬这是怎么回事

**Comment by aQingYun** [comment]
可以把报错贴出来看一下
